spec/spec_helper.rb

```diff
@@ -2,4 +2,9 @@
   config.expect_with :rspec do |c|
     c.syntax = :expect
   end
+
+  config.mock_with :rspec do |mocks|
+    mocks.syntax = :expect
```

Comment:
It's my fault, we should add only 1 new line for this: http://rspec.info/blog/2014/05/notable-changes-in-rspec-3/#partial-double-verification-configuration-option. I'll fix it shortly.